checklist_plus/perturb/base.py

```
@@ -632,3 +636,365 @@ def change_number(doc, meta=False, seed=None, n=10, skip_abbreviations=True, ent
+                if not is_brand:
+                    safe_nouns.append((token.text, token.tag_, token.i))
+                else:
+                    logging.debug(f"Skipping potential brand name: {token.text}")
+
+        if not safe_nouns:
+            logging.debug("No safe nouns found for inflection (all nouns detected as brands)")
+            return None
+
+        for _ in range(n):
+            # Pick a random noun to change
+            noun_text, tag, pos = safe_nouns[np.random.choice(len(safe_nouns))]
```

Ruff (0.14.2)

Unpacked variable `pos` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)